src/payment/models/history/dto/history.dto.ts

```diff
@@ -21,3 +29,7 @@ export class HistoryDto extends HistoryDtoBase {
 export class CoinTrackingHistoryDto extends HistoryDtoBase {
   date: number;
 }
+
+export class SimplifiedHistoryDto extends RouteHistoryDto {
+  type: HistoryTransactionType;
+}
```

Comment:
I would move this also to RouteHistoryDto and rename it to something like TypedRouteHistoryDto?